modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/configs/RequestSettings.scala

```diff
@@ -0,0 +1,12 @@
+package org.ergoplatform.dex.markets.configs
+
+import derevo.derive
+import derevo.pureconfig.pureconfigReader
+import tofu.Context
+import tofu.logging.derivation.loggable
+import scala.concurrent.duration.FiniteDuration
+
+@derive(pureconfigReader, loggable)
+case class RequestSettings(maxTimeWindow: FiniteDuration)
```

Comment:
Please keep naming consistent. We use term `Conf` instead of `Settings` across the project.